k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -750,6 +749,18 @@ private void onDiscard() {
         }
     }
 
+    private void onHighPriority(MenuItem item) {
+        if (isHighPriority) {
+            item.setTitle(getString(R.string.priority_urgent));
+            this.isHighPriority = false;
+            setTitle(getTitle().toString().split(",")[0]);
+        } else {
+            item.setTitle(getString(R.string.priority_normal));
+            this.isHighPriority = true;
+            setTitle(getTitle()+","+getString(R.string.priority_urgent));
```

Comment:
Please use:

`getString(R.string.priority_urgent, getTitle());`

and make use of formatting arguments.